addons/acr/CfgVehicles.hpp

```diff
@@ -0,0 +1,187 @@
+class CfgVehicles {
+    class O_Soldier_base_F;
+    class O_Soldier_F;
+    class UKSF_ACR_Soldier_Underwear: O_Soldier_F {
```

Comment:
General note overall, I would prefer GVAR and QGVAR be used everywhere, rather than fully resolved names written out each time. It reduces error and keeps it standard.
e.g ‘GVAR(Soldier_Underwear)’

Reply:
would this replace UKSF_ACR almost everywhere?

Reply:
Yep